### PR TITLE
Zfphil master merge

### PR DESCRIPTION
**Pull request by hmaarrfk**
I mostly just fixed the merge conflicts here. I don't actually know what this does, but it does

Closes https://github.com/dvc94ch/pykicad/pull/27

### CODE REVIEW

**Comment by dvc94ch** [comment]
Hi @hmaarrfk 

Thanks for your PRs. I added you as a collaborator.

Cheers

**Comment by hmaarrfk** [comment]
Yeah, i'm really not sure about this one. Should we just merge assuming @zfphil is a really smart guy?

File: pykicad/sexpr.py [hunk 1]
```diff
@@ -417,23 +410,46 @@ def parser(cls):
         return generate_parser(cls.tag, cls.schema)
 
     @classmethod
+<<<<<<< HEAD
```
Comment (dvc94ch) [inline pykicad/sexpr.py]: this is not right

File: pykicad/sexpr.py [hunk 2]
```diff
@@ -417,23 +410,46 @@ def parser(cls):
...
-            if not key in result:
+            if key not in result:
                 result.update(res)
             else:
                 if not isinstance(result[key], list):
                     result[key] = [result[key]]
                 result[key].append(res[key])
+<<<<<<< HEAD
         return cls(**result)
+=======
```
Comment (dvc94ch) [inline pykicad/sexpr.py]: dito

**Comment by dvc94ch** [comment]
@zfphil has write access, he could have merged the PR himself if he was done with it.

**Comment by hmaarrfk** [comment]
Hmm, ok. I'll try to see what I need from him. We are working together on some project and I think maybe I'm hitting strangeness due to his work on his own master branch.

**Pull request closed without merging**